server/test/controllers/GitHubUserTest.scala

```diff
@@ -1,35 +1,46 @@
 package controllers
 
-import akka.actor.ActorSystem
-import akka.http.scaladsl.marshalling.Marshal
-import akka.http.scaladsl.model.HttpResponse
-import akka.http.scaladsl.model.Uri.{Path, Query}
-import akka.stream.Materializer
-import controllers.Response.AccessToken
+import java.nio.file.Path
+
+import org.scalamock.scalatest.MockFactory
 import org.scalatest.DoNotDiscover
 import org.scalatestplus.play.{ConfiguredApp, PlaySpec}
+import play.api.libs.json.{JsObject, JsString}
+import play.api.libs.ws.{WSClient, WSResponse}
 import play.api.mvc.Result
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-@DoNotDiscover class GitHubUserTest extends PlaySpec with ConfiguredApp {
+@DoNotDiscover class GitHubUserTest extends PlaySpec with ConfiguredApp with MockFactory {
```

Comment:
`@DoNotDiscover` makes me cry when I want to run just one test or this **Spec** file